src/sources/aws_kinesis_firehose/mod.rs

```diff
@@ -50,6 +50,13 @@ pub struct AwsKinesisFirehoseConfig {
     #[configurable(metadata(docs::examples = "access_keys_example()"))]
     access_keys: Option<Vec<SensitiveString>>,
 
+    /// Whether or not to store the AWS Firehose Access Key in event secrets.
+    ///
+    /// If set to `true`, when incoming requests contains an Access Key sent by AWS Firehose, it will be kept in the
+    /// event secrets.
```

Comment:
```suggestion
    /// event secrets as "aws_kinesis_firehose_access_key".
```

Last nit, just want to make sure users can see where/how to access this 👍